website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,255 @@
+1. Click **Code** (at the top of the new repository’s page). Under the **Codespaces** tab,  choose **Create codespace on main**. Depending on how you've configured your computer's settings, this either opens a new browser tab with the Codespace development environment with VSCode running in it or opens a new VSCode window with the codespace in it. 
+1. Wait for the codespace to finish building by waiting for the `postCreateCommand` command to complete; this can take several minutes:
+
+    <Lightbox src="/img/codespace-quickstart/postCreateCommand.png" title="Wait for postCreateCommand to complete" />
+
+    When this command completes, you can start using the codespace development environment. The terminal the command ran in will close and you will get a prompt in a brand new terminal. 
+
+1. At the terminal's prompt, you can execute any dbt command you want. For example:
+
+    ```shell
+    /workspaces/test (main) $ dbt build
+    ```
+
+    You can also use the [duckcli](https://github.com/dbcli/duckcli) to write SQL against the warehouse from the command line or build reports in the [Evidence](https://evidence.dev/) project provided in the `reports` directory.
+    
+    For complete information, refer to the [dbt command reference](https://docs.getdbt.com/reference/dbt-commands). Common commands are:
+    
+    - [dbt compile](https://docs.getdbt.com/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+
+
+  </TabItem>
+
+</Tabs>
+
+
+
+
+
+
+
+## Generate a larger data set
+
+If you'd like to work with a larger selection of Jaffle Shop data, you can generate an arbitrary number of years of fictitious data from within your codespace. 
+
+1. Install the Python package called [jafgen](https://pypi.org/project/jafgen/). At the terminal's prompt, run:
+
+    ```shell
+    /workspaces/test (main) $ python -m pip install jafgen
+    ```
+
+1. When installation is done, run:
+    ```shell
+    /workspaces/test (main) $ jafgen --years NUMBER_OF_YEARS
+    ``` 
+    Replace `NUMBER_OF_YEARS` with the number of years you want to simulate. This command builds the CSV files and stores them in the `jaffle-data` folder, and is automatically sourced based on the `sources.yml` file and the [dbt-duckdb](/docs/core/connect-data-platform/duckdb-setup) adapter.
+
+As you increase the number of years, it takes exponentially more time to generate the data because the Jaffle Shop stores grow in size and number. For a good balance of data size and time to build, dbt Labs suggests a maximum of 6 years.
+
```

Comment:
curious if we want to add add'l steps to this quickstart guide, similar to the core install? e.g.:
- commit changes
- schedule a job
- what's next? add a confetti (optional)
-